source/frontend/index.html

```diff
@@ -171,6 +88,90 @@ <h1 id="middle-title">Recommended Recipes For You</h1>
         <recipe-upload></recipe-upload>
       </section>
     </div>
+
+    <div class="modal fade" id="signup-modal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
```

Comment:
This was fixed and merged in Shanqing's PR so just go ahead and rebase main.

Reply:
ah ok